spec/factories.rb

```diff
@@ -38,8 +46,8 @@
     row "1"
     seat "1"
     delivery_method 'electronic'
-    user_id 1
-    event_id 1
+    user
+    event
   end
```

Comment:
now when you create(:item) do don't need to first create a venue then event and pass them in, factory girl does that for you.